vistrails/gui/uvcdat/graphicsMethodsWidgets.py

```diff
@@ -504,13 +497,20 @@ def generateRanges(self):
 
             # Gen colors (Log)
             numIntervals = len(values) - 1
-            d = int(222 / (numIntervals - 1))
-            for a in range(numIntervals):
-                colors.append(16 + a * d)
             
         if self.includeZeroButtonGroup.isChecked('On'):
             values.insert(0, 0.0)
+        if str(self.ext1ButtonGroup.buttonGroup.button(self.ext1ButtonGroup.buttonGroup.checkedId()).text()).lower()[0]=="y":
```

Comment:
@doutriaux1 please wrap long lines. Long lines are very hard to read even on my high res display and could lead to bugs easily.